### ISSUE TITLE
Fitter class

### ISSUE THREAD

**Issue by ufeindt** [MickaelRigault/astrobject#41]
To assess lightcurve quality, I'll get started on a fitter class that collects the fits and makes plotting them easy. I have some code like that already from the lc simulations I ran a year ago in Berlin, just need to make it consistent with astrobject.

I'll keep the fitter separate from other objects for now because we may want to run different fitters on the same set of simulated lcs, e.g. to test photmetric typing.

**Comment by MickaelRigault** [comment]
Hi Uli, 

I left this fitter in astrobject. I think it makes more sense this way. Remark that we may want to call it LCFitter or SNLCFitter to make sure it is clear that it is actually a lightcurve fitter

Cheers

**Comment by ufeindt** [comment]
Good point, I'll change it to LCFitter I think because in principle it shouldn't be limited to SNe. (I might also add a child SNIaFitter or SALT2Fitter that has some extra stuff for Ia's)

**Comment by MickaelRigault** [comment]
sounds perfect
